### DISCUSSION TITLE
SPRINT 0 - Meet asincrónico viernes 23/09/2022 - Grupo MiniDevs

### DISCUSSION THREAD

**Discussion by veroivk** [FrancoGL/ISPC-FullStack#8] [Daily]
**SPRINT 0**
_Fecha inicio = 17/09/2022 - Fecha Fin = 03/10/2022_

Cuarta reunión. Asincrónica mediante whatsapp.

Definir qué aplicación utilizar para la idea de mapa de sitio.

**Comment by ZafkielEdt** [comment]
Las opciones fueron:

* Figma
* Wireframe

**Reply by ZafkielEdt** [reply]
La opción elegida fue Figma
